src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+
+    /** Description of task */
+    protected String description;
+    /** Boolean indicating if task is done */
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean isDone() {
```

Comment:
I thnk it's better to name a get function as what it is, for example, checkIsDone or getIsDone.